src/lib/Post.svelte

```diff
@@ -38,17 +38,16 @@
 		// Meower
 		"https://http.meower.org/",
 		"https://assets.meower.org/",
-		"https://api.meower.org/",
 		"https://forums.meower.org/",
 		// not everyone can add urls to go.meower.org, should be fine
 		"https://go.meower.org/",
 		"https://nextcloud.meower.org/",
 
-		// cubeupload
+		// Cubeupload
```

Comment:
cubeupload is the official name of the service
![image](https://user-images.githubusercontent.com/68464103/214484882-cbf20707-cef1-4f5c-9524-6eef95071520.png)

```suggestion
		// cubeupload
```